includes/admin/class-give-settings.php

```diff
@@ -651,7 +651,7 @@ public function give_settings( $active_tab ) {
 						array(
 							'id'      => 'session_lifetime',
 							'name'    => esc_html__( 'Session Lifetime', 'give' ),
-							'desc'    => esc_html__( 'Give will start a new session per user once they have donated. This option controls the lifetime a user\'s session is kept alive. An active session allows users to view donation receipts on your site without having to be logged in as long as they are using the same browser they used when donating.', 'give' ),
+							'desc'    => esc_html__( 'Control the lifetime a user\'s session is kept alive. Give will start a new session per user once they have donated. An active session allows users to view donation receipts on the site without having to be logged in.', 'give' ),

```

Comment:
The length of time a user's session is kept alive. Give starts a new session per user upon donation. Sessions allow donors to view their donation receipts without being logged in.